packages/core/src/app.ts

```diff
@@ -275,7 +275,7 @@ export namespace App {
     prefix: Schema.union([
       Schema.array(Schema.string()),
       Schema.transform(Schema.string(), (prefix) => [prefix]),
-    ] as const).description('指令前缀字符，可以是字符串或字符串数组。将用于指令前缀的匹配。'),
+    ] as const).default('').description('指令前缀字符，可以是字符串或字符串数组。将用于指令前缀的匹配。'),
```

Comment:
should be `['']` because default value will not be transformed

Reply:
Thank you for your patience in commenting on the inadequacies of the code and clarifying them.